### ISSUE TITLE
Allow setting found state for UDC

### ISSUE THREAD

**Issue by moving-bits** [cgeo/cgeo#10230]
Allow setting the found state of a UDC to either "found", "not found" (DNF) or default.

My use case is that I'm using UDC for lab caches (among other things). On having finished a lab cache I would like to see that on the map (or to hide the UDC the same way I hide other found caches). Currently I need to delete a UDC to achieve that, which also means losing the attached info (e. g. locations and answers to the stages).

Changing the found state could be put into the cache's "extras" submenu.

**Comment by Lineflyer** [comment]
Shouldn't we wait for (hopefully) have lab caches anyway?

**Comment by eddiemuc** [comment]
UDCs are in my opinion more like global waypoints and can be used for almost anything (despite the name containing the word "cache") . E.g. I can use them to mark my home, my favorite petrol station or the spot I parked my car for the days tour. Most of these waypoints do not have a "found"/"not found" semantic.

Question: recently customizable cache markers were implemented. Can't those be used for this use case? Or is this really about filtering?
